backend/app/api/routes/collections.py

```
@@ -339,15 +352,21 @@
-        collection = collection_crud.read_one(request.collection_id)
-        assistant = OpenAIAssistantCrud(client)
-        data = collection_crud.delete(collection, assistant)
-        logger.info(
-            f"[do_delete_collection] Collection deleted successfully | {{'collection_id': '{collection.id}'}}"
-        )
-        callback.success(data.model_dump(mode="json"))
+        with Session(engine) as session:
+            collection_crud = CollectionCrud(session, current_user.id)
+            collection = collection_crud.read_one(request.collection_id)
+            assistant = OpenAIAssistantCrud(client)
+            # TODO: Decouple OpenAI collection deletion from DB session handling.  
```

Ruff (0.12.2)

Trailing whitespace

Remove trailing whitespace

(W291)